modules/templates-writing-parameters.adoc

```diff
@@ -63,13 +63,44 @@ characters long consisting of all upper and lowercase alphabet letters
 and numbers.
 
 The syntax available is not a full regular expression syntax. However, you can
-use `\w`, `\d`, and `\a` modifiers:
+use `\w`, `\d`, `\a`, and `\A` modifiers:
 
 - `[\w]{10}` produces 10 alphabet characters, numbers, and underscores. This
 follows the PCRE standard and is equal to `[a-zA-Z0-9_]{10}`.
 - `[\d]{10}` produces 10 numbers. This is equal to `[0-9]{10}`.
 - `[\a]{10}` produces 10 alphabetical characters. This is equal to
 `[a-zA-Z]{10}`.
+- `[\A]{10}` produces 10 punctuation or symbol characters. This is equal to ``[~!@#$%^&*()-_+={}[]\|<,>.?/"';:`]{10}``.
+
+[NOTE]
+====
+Depending on if the template is written in YAML or JSON, and the type of string that the modifier is embedded within, you might need to escape the backslash with a second backslash. The following examples are equivalent:
+
+.Example YAML template with a modifier
+[source,yaml]
+----
+  parameters:
+  - name: singlequoted_example
+    generate: expression
+    from: '[\A]{10}'
+  - name: doublequoted_example
+    generate: expression
+    from: "[\\A]{10}"
+----
+
+.Example JSON template with a modifier
+[source,json]
+----
+{
+    {"parameters": [
+       {"name": "json_example",
+        "generate": "expression",
+        "from": "[\\A]{10}"
+       }
+    ]
+}
+----
+====
```

Comment:
This note look good. I think that there is an extra `{` at the beginning of the JSON example though. So:

`{"parameters": [
       {"name": "json_example",
        "generate": "expression",
        "from": "[\\A]{10}"
       } 
   ]
}`